.github/workflows/maven.yml

```diff
@@ -0,0 +1,18 @@
+name: Build and package
```

Comment:
what is the purpose in one more github action? There is one for PR's (see pr.yml) and one for building from master/tags. Both of them include maven build

Reply:
The purpose is to have maven build only to validate wip branches, it seems tests aren't windows compatible so it's the only way for me to validate changes.

Reply:
It should work on windows just fine, will check what is there. Getting back to workflow - I'm concerned that it duplicates existing stuff in some way. Will check how to filter events to avoid that